doc/website/site.webmanifest

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "XDG Desktop Portal",
+    "short_name": "XDG Desktop Portal",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff"
```

Comment:
Is this correct, actually?

Reply:
Probably not, but it also probably doesn't matter. The Theme color might.

Still, tried to see how this looks and currently, it doesn't even find the images above.

Reply:
Should be fixed. Please let me know, if I should rebase and squash these to the relevant commit.

![image](https://github.com/flatpak/xdg-desktop-portal/assets/5943908/641e0a86-8f04-47e4-a966-e04a8ef493b5)